src/components/motion/MotionController.cpp

```diff
@@ -14,69 +14,87 @@ void MotionController::Update(int16_t x, int16_t y, int16_t z, uint32_t nbSteps)
   this->x = x;
   this->y = y;
   this->z = z;
-  int32_t deltaSteps = nbSteps - this->nbSteps;
+  uint32_t deltaSteps = nbSteps - this->nbSteps;
   this->nbSteps = nbSteps;
   if (deltaSteps > 0) {
     currentTripSteps += deltaSteps;
   }
 }
 
-bool MotionController::Should_RaiseWake(bool isSleeping) {
-  if ((x + 335) <= 670 && z < 0) {
+bool MotionController::Should_RaiseWake() {
+  bool isSleeping = true;
```

Comment:
`isSleeping` isn't always true. Maybe add the `isSleeping` parameter to the callback function as well?

Reply:
I did that as a placeholder since I knew I would need to refactor it. I think it makes sense to pull the logic for setting the tilt threshold into MotionController::Update. In my opinion, MotionController::Should_RaiseWake should only be called when sleeping.